docker/postgres-setup/init.sql

```diff
@@ -0,0 +1,31 @@
+-- create datahub database
+CREATE DATABASE IF NOT EXISTS DATAHUB_DB_NAME;
+
+-- create metadata aspect table
+CREATE TABLE IF NOT EXISTS DATAHUB_DB_NAME.metadata_aspect_v2 (
+  urn                           varchar(500) not null,
+  aspect                        varchar(200) not null,
+  version                       bigint not null,
+  metadata                      text not null,
+  systemmetadata                text,
+  createdon                     timestamp not null,
+  createdby                     varchar(255) not null,
+  createdfor                    varchar(255),
+  CONSTRAINT pk_metadata_aspect_v2 PRIMARY KEY (urn, aspect, version)
+);
+
+INSERT INTO DATAHUB_DB_NAME.metadata_aspect_v2 (urn, aspect, version, metadata, createdon, createdby) VALUES(
```

Comment:
Can you follow what is done here? https://github.com/linkedin/datahub/blob/master/docker/mysql-setup/init.sql#L19 Ensures that duplicate rows are never added. You can ignore the metadata_index creation.

Reply:
ad31985

just added the insert if not exists query^^